Jenkinsfile

```diff
@@ -0,0 +1,7 @@
+// Readme @ http://gitlab.iex.ec:30000/iexec/jenkins-library
+@Library('global-jenkins-library@1.6.1') _
+
+def nativeImage = buildSimpleDocker_v2(dockerfileDir: '.', buildContext: '.',
+        dockerImageRepositoryName: 'private-data-email-app', imageprivacy: 'public')
```

Comment:
I think we can take this opportunity to rename the docker image name to `confidential-email-sender` or other.

Reply:
I don't think the app is automatically deployed as of now. It wasn't even automatically build before I added this Jenkinsfile.

Reply:
Don't you want to rename `image` names?

Reply:
Yes we are thinking about using the name `confidential-email-sender` instead of `private-data-email-app`.

Reply:
Sure, see https://github.com/iExecBlockchainComputing/confidential-email-sender/pull/2/commits/ed7d8160996d70f9e3b22ee90fadb0e60e438708.